src/main/java/com/liferay/faces/test/selenium/IntegrationTesterBase.java

```diff
@@ -127,6 +127,6 @@ protected final void signIn(Browser browser, String signInURL, String loginXpath
 		WebElement loginElement = browser.findElementByXpath(loginXpath);
 		browser.click(signInButtonXpath);
 		browser.waitUntil(ExpectedConditions.stalenessOf(loginElement));
-		browser.waitForElementVisible("//body");
+		browser.waitForElementDisplayed("//body");
```

Comment:
@ngriffin7a, I disagree. It's reasonable to consider the `<body>` element visible or displayed based on similar criteria for a `<div>`. A `<div>` element has no visible aspects by itself, it's only truly visible if it has visible content and its CSS `display: != 'none'` (oversimplifying but the point stands). The same goes for `<body>`. You can see [here that `style="display: none;"` can be used on a `<body>` tag to hide it](http://jsbin.com/tiwipewimi/edit?html,output). That implies that it is displayed by default.